addons/binding/org.openhab.binding.chromecast/ESH-INF/thing/thing-types.xml

```diff
@@ -106,10 +236,209 @@
         <description>Plays a given URI</description>
     </channel-type>
 
-    <channel-type id="control">
-        <item-type>Player</item-type>
-        <label>Control</label>
-        <description>Player control</description>
+    <channel-type id="onlinestatus" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Online</label>
+        <description>Is the Chromecast online</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <!--App Information -->
+    <channel-type id="idling" advanced="true">
+        <item-type>Switch</item-type>
+        <label>Idling</label>
+        <description>Is Chromecast active or idling</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="appName" advanced="true">
+        <item-type>String</item-type>
+        <label>App</label>
+        <description>Name of the currently running application</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="appId" advanced="true">
+        <item-type>String</item-type>
+        <label>App Id</label>
+        <description>Id of the currently running application</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="statustext" advanced="true">
+        <item-type>String</item-type>
+        <label>App Status</label>
+        <description>Status reported by the current application</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <!-- Media Information -->
+    <channel-type id="currentTime" advanced="true">
+        <item-type>Number</item-type>
+        <label>Current Time</label>
+        <description>Current time of currently playing media</description>
```

Comment:
It is in seconds. Is this something we'd normally round to the nearest second or do you let the end-user (sitemap/whatever) do that?